### PR TITLE
adding new images for meeting key

### PR DESCRIPTION
**Pull request by chriswedgwood**

### CODE REVIEW

**Comment by chriswedgwood** [comment]
[ICONS for filtering Physical + ZOOM](https://trello.com/c/oVGewINT/145-icons-for-filtering-physical-zoom)

File: aalondon/static/js/components/MeetingSearchForm.js [hunk 1]
```diff
@@ -43,57 +43,58 @@ class MeetingSearchForm extends React.Component {
...
   handleClearFilters() {
-    
+
     this.props.onClearFilters();
-  } 
+  }
 
-  handleMeetingTypeChange(eventKey,e){
-   this.props.onMeetingTypeChange(e.value);
+  handleMeetingTypeChange(eventKey, e) {
```
Comment (tintinnabulate) [inline aalondon/static/js/components/MeetingSearchForm.js:96]: I'm a big fan of code tidy-ups, but mixing them with feature implementations in the same commit makes it tricky for me to do code-reviews.

In future, could you please split code tidy-ups & feature implementations into dedicated commits? i.e. a feature impl. should have it's own commit without code tidy ups.

**Review by tintinnabulate — APPROVED** [review]
For some reason this PR hasn't triggered a preview on staging.aa-london.com - I thought PRs were supposed to push to staging. Are the Github Actions working?

Other than that point, it seems okay - I haven't done a local build to check how it looks, so it's a rather lightweight review.

File: aalondon/static/js/components/MeetingSearchForm.js (hunk 1 above)
Comment (chriswedgwood) [reply aalondon/static/js/components/MeetingSearchForm.js:96]: Yes I must have done a format doc in vscode

**Review by chriswedgwood — COMMENTED** [review]
> For some reason this PR hasn't triggered a preview on staging.aa-london.com - I thought PRs were supposed to push to staging. Are the Github Actions working?
> 
> Other than that point, it seems okay - I haven't done a local build to check how it looks, so it's a rather lightweight review.

Once the pr is merged into master it deploys to staging

**Pull request merged**